graphql/integration_atomic_test.go

```diff
@@ -89,7 +94,23 @@ func runTestsInDirectory(t *testing.T, state atomicGraphQLState) {
 	err = json.Unmarshal(resultsFile, &tests)
 	require.NoError(t, err)
 
-	require.NoError(t, setupData(*evergreen.GetEnvironment().DB(), testData))
+	// Delete exactly the documents added to the task_logg coll instead of dropping task log db
+	// we do this to minimize deleting data that was not added from this test suite
+	if testData[state.taskLogColl] != nil {
+		logsDb := evergreen.GetEnvironment().Client().Database(state.taskLogDB)
+		idArr := []string{}
+		var docs []model.TaskLog
+		catcher := grip.NewBasicCatcher()
```

Comment:
You aren't doing anything with the caught errors. I think you want to fail the test immediately in case of error (using require) but you don't actually need a catcher for that